docs/releases/1.21-NOTES.md

```diff
@@ -43,11 +39,11 @@ See the [warm pool](https://kops.sigs.k8s.io/instance_groups/#warmpool-aws-only)
 * Support for Kubernetes versions 1.15 and 1.16 are deprecated and will be removed in kOps 1.22.
 * Support for launch configurations has been removed in favour of launch templates.
 
-# Full change list since 1.21.0 release
+# Full change list since 1.20.0 release
```

Comment:
No, this is correct. This is the changes to kOps since 1.20.0, which is what one would expect in the 1.21 release notes.